GitUI/CommandsDialogs/SettingsDialog/Pages/SortingSettingsPage.Designer.cs

```diff
@@ -54,16 +55,17 @@ private void InitializeComponent()
             // tlpnlMain
             // 
             tlpnlMain.ColumnCount = 1;
-            tlpnlMain.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
-            tlpnlMain.Controls.Add(this.gbGeneral, 0, 0);
-            tlpnlMain.Dock = System.Windows.Forms.DockStyle.Fill;
-            tlpnlMain.Location = new System.Drawing.Point(8, 8);
+            tlpnlMain.ColumnStyles.Add(new ColumnStyle());
+            tlpnlMain.Controls.Add(gbGeneral, 0, 0);
+            tlpnlMain.Dock = DockStyle.Fill;
+            tlpnlMain.Location = new Point(8, 8);
             tlpnlMain.Name = "tlpnlMain";
             tlpnlMain.RowCount = 3;
-            tlpnlMain.RowStyles.Add(new System.Windows.Forms.RowStyle());
-            tlpnlMain.RowStyles.Add(new System.Windows.Forms.RowStyle(System.Windows.Forms.SizeType.Percent, 100F));
-            tlpnlMain.RowStyles.Add(new System.Windows.Forms.RowStyle(System.Windows.Forms.SizeType.Absolute, 10F));
-            tlpnlMain.Size = new System.Drawing.Size(1565, 1339);
+            tlpnlMain.RowStyles.Add(new RowStyle());
+            tlpnlMain.RowStyles.Add(new RowStyle(SizeType.Percent, 100F));
+            tlpnlMain.RowStyles.Add(new RowStyle(SizeType.Absolute, 10F));
+            tlpnlMain.Size = new Size(1511, 669);
+            tlpnlMain.TabIndex = 0;
```

Comment:
remove tabindex?

Reply:
The designer always emits TabIndex, however, its value often at odds with the "natural" tab order (used by accessibility tools, it's derived from order of .Add() of the container control). It's something I aspired to fix but never had time for...

So it's better to manually remove TabIndex.